project/plugins/iam.py

```diff
@@ -330,9 +378,35 @@ def update_user_password(pw):
 
 
 def get_ssm_client(configMap, **key_args):
-    if values.profile is not None:
-        session = get_iam_session()
+    if key_args.get('region') is not None:
+        region_name = key_args.get('region')
```

Comment:
Same deal here - probably better to just use a default value in key_args.get instead of the if/else block